### PR TITLE
Add missing comments in k8s.io/api/core/v1

### PR DESCRIPTION
**Pull request by thockin**
Supersedes https://github.com/kubernetes/kubernetes/pull/125233

/kind cleanup
```release-note
NONE
```

### CODE REVIEW

File: staging/src/k8s.io/api/core/v1/types.go [hunk 1]
```diff
@@ -2939,36 +2946,35 @@ type Container struct {
...
-	// Exec specifies the action to take.
+	// Exec specifies a command to execute in the container.
 	// +optional
 	Exec *ExecAction `json:"exec,omitempty" protobuf:"bytes,1,opt,name=exec"`
-	// HTTPGet specifies the http request to perform.
+	// HTTPGet specifies an HTTP GET request to perform.
 	// +optional
 	HTTPGet *HTTPGetAction `json:"httpGet,omitempty" protobuf:"bytes,2,opt,name=httpGet"`
-	// TCPSocket specifies an action involving a TCP port.
+	// TCPSocket specifies an connection to a TCP port.
```
Comment (mrunalp) [inline staging/src/k8s.io/api/core/v1/types.go:2955]: nit:  a connection to

Comment (thockin) [reply staging/src/k8s.io/api/core/v1/types.go]: Dangit.  Fixed.

**Review by mrunalp — COMMENTED** [review]
/lgtm

**Comment by pacoxu** [comment]
/milestone v1.32
approved before code freeze

**Comment by pacoxu** [comment]
CI shows that `Please run 'hack/update-openapi-spec.sh'`.

**Pull request merged**